docs/conf/writing.md

```diff
@@ -67,15 +67,33 @@ production build.
 
 ## Configuration Files
 
-It is possible to have multiple configuration files (bundled and/or remote),
+It is possible to have multiple configuration files (bundled and/or remote\*),
 both to support different environments, but also to define configuration that is
 local to specific packages. The configuration files to load are selected using a
 `--config <local-path|url>` flag, and it is possible to load any number of
 files. Paths are relative to the working directory of the executed process, for
 example `package/backend`. This means that to select a config file in the repo
 root when running the backend, you would use `--config ../../my-config.yaml`,
 and for config file on a config server you would use
-`--config https://some.domain.io/app-config.yaml`
+`--config https://some.domain.io/app-config.yaml`<br/>
+
+**\*Note**: In order to use remote urls, ensure that the option 'remote' is
```

Comment:
Do you think we could slim this down a little bit? Perhaps mentioning above that when URLs are passed you also need to set the `remote` option in the `loadBackendConfig` call? And let the rest if it be discovered. Bit weary of this config section already being quite long and I think it's best if it's a pretty lightweight read